omero/sysadmins/unix/walkthrough/walkthrough_debian9.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -e -u -x
+source settings.env
+
+#start-step01: As root, install dependencies
+
+apt-get update
+
+# installed for convenience
+apt-get -y install unzip wget bc
+
+# install Java
+apt-get -y install openjdk-8-jre-headless
+
+# install dependencies
+
+apt-get -y install \
+	python-{pip,pillow,numpy,scipy,tables,virtualenv,yaml,jinja2}
+
+pip install --upgrade pip
+# install Ice
+#start-recommended-ice
+apt-get -y install libssl-dev libbz2-dev libmcpp-dev libdb++-dev libdb-dev libdb-java
+apt-get -y install zeroc-ice-all-runtime
+pip install "zeroc-ice>3.5,<3.7"
+#end-recommended-ice
+
+
+# install Postgres
+apt-get -y install postgresql
+service postgresql start
```

Comment:
I doubt that something will already be available for debian 9 already
The change mentioned above will imply that it will work in docker 
but won't show up in the doc